apps/cowswap-frontend/src/modules/swap/pure/Row/RowFeeContent/index.tsx

```diff
@@ -37,7 +37,7 @@ export function RowFeeContent(props: RowFeeContentProps) {
     <StyledRowBetween {...styleProps} alignContentRight={showFiatOnly}>
       {!noLabel && (
         <RowFixed>
-          <TextWrapper>{swapZeroFee ? 'Est. fees' : 'Fees'}</TextWrapper>
+          <TextWrapper>{label}</TextWrapper>
```

Comment:
Ah, that's where that change is coming from! Disregard the first comment then.

Reply:
👍